common/src/main/java/io/homeassistant/companion/android/common/data/websocket/impl/WebSocketRepositoryImpl.kt

```diff
@@ -179,6 +180,17 @@ class WebSocketRepositoryImpl @Inject constructor(
         }
     }
 
+    override suspend fun getConversation(speech: String): ConversationResponse? {
```

Comment:
For future PR, you get a `conversastion_id` back (`str | null`) in `ConversationResponse`. As long as the activity is open, it would be nice if we can pass that back in. Some conversation agent in the future will be able to do dialogues.

Reply:
I will add a `TODO` to keep track of this request :)